imap_processing/cdf/config/shared/default_global_cdf_attrs_schema.yaml

```diff
@@ -69,6 +69,14 @@ Discipline:
   required: true
   validate: true
   overwrite: false
+File_naming_convention:
```

Comment:
I was also going to add this to the schema but saw that @maxinelasp created a ticket related to this. Issue #594